File: src/System.Windows.Forms.Analyzers.CSharp/tests/UnitTests/ImplementITypedDataObjectInAdditionToIDataObjectAnalyzer/Data/UnrelatedIDataObject.cs
using System;
using System.Runtime.Versioning;

[assembly: TargetFramework(".NETCoreApp,Version=v10.0", FrameworkDisplayName = ".NET 10.0")]

namespace System.Windows.Forms.Analyzers.CSharp.Tests.ImplementITypedDataObjectInAdditionToIDataObjectAnalyzer.Data
{
    internal interface IDataObject
    {
        void DoStuff();
    }

    internal class UnrelatedIDataObject : IDataObject
    {
        public void DoStuff() => throw new NotImplementedException();
    }
}

File: src/System.Windows.Forms.Analyzers.VisualBasic/tests/UnitTests/System.Windows.Forms.Analyzers.VisualBasic.Tests/WFO1001/Data/UnrelatedIDataObject.vb
Imports System
Imports System.Runtime.Versioning

<Assembly: TargetFramework(".NETCoreApp,Version=v10.0", FrameworkDisplayName:=".NET 10.0")>

Namespace System.Windows.Forms.Analyzers.VisualBasic.Tests.ImplementITypedDataObjectInAdditionToIDataObjectAnalyzer.Data

    Friend Interface IDataObject
        Sub DoStuff()
    End Interface

    Friend Class UnrelatedIDataObject
        Implements IDataObject

        Public Sub DoStuff() Implements IDataObject.DoStuff
            Throw New NotImplementedException()
        End Sub

    End Class

End Namespace
